stable/prometheus/README.md

```diff
@@ -239,14 +242,14 @@ Parameter | Description | Default
 `server.persistentVolume.enabled` | If true, Prometheus server will create a Persistent Volume Claim | `true`
 `server.persistentVolume.accessModes` | Prometheus server data Persistent Volume access modes | `[ReadWriteOnce]`
 `server.persistentVolume.annotations` | Prometheus server data Persistent Volume annotations | `{}`
-`server.persistentVolume.existingClaim` | Prometheus server data Persistent Volume existing claim name | `""`
```

Comment:
How do you manage reusing existing storage now ?

Reply:
My bad just added it back.

Reply:
In fact, it should have been removed, and now I've done so (see comments below). Since the server and alertmanager pods will now be managed by `StatefulSets`, two things will happen: Helm won't delete `PVCs` when the release is deleted, and the pods will reuse these PVCs if they exist.